bin/cli.js

```diff
@@ -20,7 +20,7 @@ var config = {
   defaults: [
     { name: 'dir', abbr: 'd', help: 'set the directory for Dat' },
     { name: 'logspeed', default: 400 },
-    { name: 'port', default: 3282, help: 'port to use for connections' },
+    { name: 'port', default: 0, help: 'port to use for connections (tries 3282 by default)' },
```

Comment:
I think we'd be better removing this default. Otherwise `dat-node` won't try 3282 first.

Reply:
Oh no it'll try 3828 first, nm. But still not sure we need this, just confuses things =).